packages/manager/apps/dedicated/client/app/account/identity-documents/user-identity-documents.controller.js

```diff
@@ -94,6 +92,10 @@ export default class AccountUserIdentityDocumentsController {
     this.trackPage(TRACKING_TASK_TAG.uploadError);
   }
 
+  gotToDashboard() {
+    this.trackClick(TRACKING_TASK_TAG.gotToHub);
+  }
```

Comment:
This function does not do what its name suggests. Please reconsider its name or include the goTo function in its body